cmd/podman/play/kube.go

```diff
@@ -144,12 +147,54 @@ func kube(cmd *cobra.Command, args []string) error {
 		}
 		kubeOptions.StaticMACs = append(kubeOptions.StaticMACs, m)
 	}
+	if kubeOptions.Down {
+		return teardown(yamlfile)
+	}
+	return playkube(yamlfile)
+}
 
-	report, err := registry.ContainerEngine().PlayKube(registry.GetContext(), yamlfile, kubeOptions.PlayKubeOptions)
+func teardown(yamlfile string) error {
+	var (
+		podStopErrors utils.OutputErrors
+		podRmErrors   utils.OutputErrors
+	)
+	reports, err := registry.ContainerEngine().PlayKubeDown(registry.GetContext(), yamlfile, kubeOptions.PlayKubeOptions)
 	if err != nil {
 		return err
 	}
 
+	// Output stopped pods
+	fmt.Println("Pods stopped:")
+	for _, stopped := range reports.StopReport {
+		if len(stopped.Errs) == 0 {
+			fmt.Println(stopped.Id)
+		} else {
+			podStopErrors = append(podStopErrors, stopped.Errs...)
+		}
+	}
+	// Dump any stop errors
+	lastStopError := podStopErrors.PrintErrors()
+	if lastStopError != nil {
+		fmt.Fprintf(os.Stderr, "Error: %s\n", lastStopError)
+	}
+
+	// Output rm'd pods
+	fmt.Println("Pods removed:")
+	for _, removed := range reports.RmReport {
+		if removed.Err == nil {
+			fmt.Println(removed.Id)
+		} else {
+			podRmErrors = append(podRmErrors, removed.Err)
+		}
+	}
+	return podRmErrors.PrintErrors()
```

Comment:
Ditto to above comment about failed removed/stopped pods

Reply:
yup, same answer